render/src/main/java/org/codice/imaging/nitf/render/imagerep/ImageRepresentationHandler.java

```diff
@@ -48,7 +52,12 @@ void renderPixelBand(DataBuffer dataBuffer, int pixelIndex, ImageInputStream ima
      */
     BufferedImage createBufferedImage(int width, int height);
 
-    void applyPixelMask(DataBuffer data, int pixelIndex);
-
-    void renderPadPixel(DataBuffer data, int pixelIndex);
+    /**
+     * Apply image mask to one pixel within a block.
+     *
+     * @param imageMask the image mask information
+     * @param data the data buffer (corresponding the BufferedImage) to mask
+     * @param pixel the pixel index within the block to operate on
+     */
+    public void renderPadPixel(ImageMask imageMask, DataBuffer data, int pixel);
```

Comment:
For consistency, we should probably call the last parameter 'pixelIndex' instead of 'pixel'.  When I first looked at the method signature, I thought this was referring to the pixel data.